sequence.go

```diff
@@ -92,3 +94,31 @@ func getFeatureSequence(feature Feature, location Location) string {
 
 	return sequenceString
 }
+
+
+//RandomProteinSequence returns a random protein sequence as a string that have size n, starts with aminoacid M and finishes with aminoacid *. The random generator uses the seed provided as parameter.
+func RandomProteinSequence(n int, seed int64) (string, error) {
+
+        if n <= 2 {
+            err := errors.New("The parameter n should be higher than 2, please select another n in RandomProteinSequence")
+            return "", err
+        }
+
+        var aminoAcidsAlphabet = []rune("ACEDGFIHKLNQPSRTWVY")
+        rand.Seed(seed)
+
+        randomSequence := make([]rune, n)
+
+        for i := range randomSequence {
+            if (i == 0) {
+                randomSequence[i] = 'M'
```

Comment:
What is M? can just say it's a start codon and include wiki link.

What is *? Can just say stop codon and include wiki link.